common-text-blocks.md

```diff
@@ -0,0 +1,35 @@
+
+# Reusable reference blocks
+
+This section is for reusable blocks to establish standards for presentation
+within our Documentation in regards to our blocks of warnings and other
+common areas. These use Vuepress' slots for danger, warning, tip and details.
+Use the first two as sparingly as possible unless ***ABSOLUTELY*** necessary.
+
+
+## For sections that are under construction.
+
+``` ::: warning Under Construction
+
+The documentation here is under heavy construction. Updates are coming, and
+likely in the works. If not, please report a issue in our repository for this
+site. You can find on the page linked in our header.
+
+:::
+```
+
+
+## For archived sections
+
+```
+
+::: danger STOP
+
+This is an archive of the atom documentation. This is only here for historical
+purposes. This is solely a functional reference of how things used to work.
+Processes in this section may have changed in both functionality and
+presentation since these were originally written. Our updated documentation can
+be found at: <insert link here>
+
+:::
```

Comment:
My suggestion would be:

```suggestion
::: danger STOP

This is being kept for archival purposes only from the original Atom documentation. As this may no longer be relevant to Pulsar you use this at your own risk.
Current Pulsar documentation is found at [Generic Link](link/to/main/docs/section).

:::
```

The link wouldn't be specific to link to newer versions etc., it would just take you away from the archive area and into the main docs - for example for people who landed on that page from a google search.

Reply:
Remove the extra spaces, Not sure why, but it doesn't like blank newlines mixed in with these. Found that out the hard way with my original versions.